src/Bicep.LangServer/Deploy/DeploymentHelper.cs

```diff
@@ -17,25 +17,11 @@
 
 namespace Bicep.LanguageServer.Deploy
 {
-    public class DeploymentHelper
+
+    public class DeploymentHelper : IDeploymentHelper
     {
-        /// <summary>
-        /// Starts a deployment at provided target scope and returns <see cref="BicepDeploymentStartResponse"/>.
-        /// </summary>
-        /// <param name="deploymentCollectionProvider">deployment collection provider</param>
-        /// <param name="armClient">arm client</param>
-        /// <param name="documentPath">path to bicep file used in deployment</param>
-        /// <param name="template">template used in deployment</param>
-        /// <param name="id">id string to create the ResourceIdentifier from</param>
-        /// <param name="scope">target scope</param>
-        /// <param name="location">location to store the deployment data</param>
-        /// <param name="deploymentId">deployment id</param>
-        /// <param name="portalUrl">azure management portal URL</param>
-        /// <param name="deploymentName">deployment name</param>
-        /// <param name="parametersFileContents">contents of parameter file used in deployment</param>
-        /// <param name="deploymentOperationsCache">deployment operations cache that needs to be updated</param>
-        /// <returns><see cref="BicepDeploymentStartResponse"/></returns>
-        public static async Task<BicepDeploymentStartResponse> StartDeploymentAsync(
+        /// <inheritdoc/>
```

Comment:
Sorry, I am unfamiliar with inheritdoc. Does it retrieve the param definitions from a different file?

Reply:
Yes, it will retrieve info from `IDeploymentHelper`